### PR TITLE
Features/push docker hub

### PR DESCRIPTION
**Pull request by BRuslanB**
The Docker App container was pushed on Docker Hub and added to docker-compose.yml

### CODE REVIEW

**Comment by BRuslanB** [comment]
Now the application and the database can be run through docker-compose up ;)

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM openjdk:17-alpine
+MAINTAINER BRuslan
+COPY build/libs/family-budget-0.0.1-SNAPSHOT.jar docker-family-budget-0.0.1.jar
+ENTRYPOINT ["java", "-jar", "docker-family-budget-0.0.1.jar"]
```
Comment (btimur) [inline Dockerfile:4]: restrict run container under sepcific user
USER  1000

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+FROM openjdk:17-alpine
+MAINTAINER BRuslan
+COPY build/libs/family-budget-0.0.1-SNAPSHOT.jar docker-family-budget-0.0.1.jar
```
Comment (btimur) [inline Dockerfile:3]: mkdir app
copy to app
cd app

Comment (btimur) [reply Dockerfile:3]: optiize image using sprrinboot docker layer

**Pull request merged**